src/client/app/utils/translate.ts

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import store from '../index';
+import { addLocaleData, IntlProvider, defineMessages } from 'react-intl';
+import * as en from 'react-intl/locale-data/en';
+import * as fr from 'react-intl/locale-data/fr';
+import * as localeData from '../translations/locales/data.json';
+
+export default function translate(messageID: string): string {
```

Comment:
This feels very heavyweight for something that runs each time a string gets translated. Can you cache most of this on the first run?

Reply:
You also might want to export it with a shorter name like `t` so it doesn't add a ton of line length when used. Django does something similar with its translations.

Reply:
`t` is frequently used throughout Redux to denote types